_posts/2021-02-01-launching-the-solidity-forum.md

```diff
@@ -0,0 +1,77 @@
+
+#### → Solidity Compiler
+[“Solidity Compiler”](https://forum.soliditylang.org/c/solidity-compiler/6) is a category for discussions and feedback about the Solidity compiler.
+
+⚠️ Please do not use the forum to report issues. Please report issues directly via the [Solidity Issue Tracker](https://github.com/ethereum/solidity/issues/new/choose) in Github.
+
+#### → Language Design
+[“Language Design”](https://forum.soliditylang.org/c/language-design/9) is the dedicated place for proposals and discussion on new language features and their implementation in the early stages of ideation. This is the place to discuss the soundness of proposals and define them further. As soon as proposals get more tangible their implementation will also be discussed in the Solidity GitHub organisation in the form of GitHub issues.
+
+Examples of topics that could fit the Language Design category:
+
+- "Change the way inheritance works"
+- "Introduce a ``switch`` statement" 
+
+ℹ️ You can follow the implementation status of new features in the [Solidity Github project](https://github.com/ethereum/solidity/projects/43). Issues in the design backlog need further specification and will either be discussed in a language design call or in a regular team call. You can see the upcoming changes for the next breaking release by changing from the default branch (develop) to the [breaking branch](https://github.com/ethereum/solidity/tree/breaking).
+
+#### → Tools & Infrastructure
+[“Tools & Infrastructure”](https://forum.soliditylang.org/c/tooling-infrastructure/10) is the category for meta discussion about developer tooling, e.g. IDEs, debuggers, development frameworks, security tools, and infrastructure with a focus on the interactions and improvements between tooling and Solidity.
+
+ℹ️ This category is not meant for reporting bugs in tools or proposing new tooling features. To do that, please refer to the respective GitHub organisations of the individual projects.
+
+#### → Documentation
+Meta discussion about the documentation and its translations can take place in the [“Documentation”](https://forum.soliditylang.org/c/documentation/8) category.
+
+⚠️ Again, please do not use the forum to report issues (e.g. typos or broken links in the documentation). Please report issues directly via the [Solidity Issue Tracker](https://github.com/ethereum/solidity/issues/new/choose) in Github.
+
+#### → Code Wizards
+In ["Code Wizards"](https://forum.soliditylang.org/c/code-wizards/7) you can share useful tips or code snippets you came up with that are worth spreading. You can also discuss experimental Solidity implementations and get feedback on it.
+
+ℹ️ The Solidity forum will not the place for ad-hoc support questions. For urgent Solidity support questions, please use the [Solidity Gitter chat](https://gitter.im/ethereum/solidity) or consider checking out the [Ethereum StackExchange](https://ethereum.stackexchange.com/).
```

Comment:
Something like: It's also no place for audit requests, although it is the right place to ask for feedback on a specific mechanism or construct.